brainstormforce/ultimate-addons-for-gutenberg
# Fix: Border styling issues in multiple blocks on both the side.

### Description
Fix Border styling issues in multiple blocks on both sides.
I have updated selectors for responsive devices for border issues in this PR.
I fixed this responsive issue in multiple blocks.
1. Multi-button
2. Info-box
3. Post grid
4. CTA
5. Forms
6. Marketing button

### Screenshots
Testing Evidence - https://share.getcloudapp.com/6quNLEEY

### Types of changes
Bug fix (non-breaking change which fixes an issue) 

### How has this been tested?
1. Switch to next-release
2. Drag and drop blocks
3. Apply all border stylings for all the responsive devices
4. Check the backend and front-end it is working or not
5. Switch to the branch mentioned in this PR
6. check border stylings 

### Checklist:
